docs/native-node-modules.md

```diff
@@ -0,0 +1,59 @@
+# Native Node Modules
+
+For some features, the desktop version of Element can make use of native Node
+modules. These allow Riot to integrate with the desktop in ways that a browser
+cannot.
+
+While native modules enable powerful new features, they must be complied for
+each operating system. For official Element releases, we will always build these
+modules from source to ensure we can trust the compiled output. In the future,
+we may offer a pre-compiled path for those who want to use these features in a
+custom build of Element without installing the various build tools required.
+
+Do note that compiling a module for a particular operating system
+(Linux/macOS/Windows) will need to be done on that operating system.
+Cross-compiling from a host OS for a different target OS may be possible, but
+we don't support this flow with Element dependencies at this time.
+
+At the moment, we need to make some changes to the Element release process before
+we can support native Node modules at release time, so these features are
+currently disabled by default until that is resolved. The following sections
+explain the manual steps you can use with a custom build of Element to enable
+these features if you'd like to try them out.
```

Comment:
Perhaps we should add a sentence to be more explicit about why this is different from the main steps listed on the README of this desktop repo...? The (older) steps in this doc are effectively a bespoke, manual install process that doesn't consider all the details building native bits in automation.